src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.ui;
+
+public class Formatter {
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+    
+    public static String getLinePrefix() {
```

Comment:
This class should do a bit more than just storing special formatting strings. As we call it the formatter, it should format the string for us, i.e. take in a raw string, then format accordingly by adding separator, etc and return the formatted string to user.